Binary files ./misc/seqan_instrumentation/last_revision_copy/core/include/seqan/seeds2/.seeds_extension.h.swp and ./core/include/seqan/seeds2/.seeds_extension.h.swp differ
Binary files ./misc/seqan_instrumentation/last_revision_copy/sandbox/workshop/apps/ministellar/.ministellar.cpp.swp and ./sandbox/workshop/apps/ministellar/.ministellar.cpp.swp differ
diff -u -r -N -x '*.o' -x Thumbs.db -x .DS_Store -x CMakeCache.txt -x misc/seqan_instrumentation/userdata/id.txt -x /Users/konrad/Projects/workshop11/seqan/misc/seqan_instrumentation/userdata/id.txt -x misc/seqan_instrumentation/userdata/rfwo8250x9sgr52s_stats.txt -x /Users/konrad/Projects/workshop11/seqan/misc/seqan_instrumentation/userdata/rfwo8250x9sgr52s_stats.txt -x .svn -x build -x util -x misc -x docs -x misc/seqan_instrumentation/bin -x /Users/konrad/Projects/workshop11/seqan/misc/seqan_instrumentation/bin -x misc/seqan_instrumentation/last_revision_copy -x /Users/konrad/Projects/workshop11/seqan/misc/seqan_instrumentation/last_revision_copy -x misc/seqan_instrumentation/userdata -x /Users/konrad/Projects/workshop11/seqan/misc/seqan_instrumentation/userdata ./misc/seqan_instrumentation/last_revision_copy/sandbox/workshop/apps/ministellar/ministellar.cpp ./sandbox/workshop/apps/ministellar/ministellar.cpp
--- ./misc/seqan_instrumentation/last_revision_copy/sandbox/workshop/apps/ministellar/ministellar.cpp	2011-09-13 12:09:43.000000000 +0200
+++ ./sandbox/workshop/apps/ministellar/ministellar.cpp	2011-09-13 12:13:03.000000000 +0200
@@ -108,7 +108,7 @@
         TSeed seed(
             clippedBeginPosition(row(align, 0)), clippedBeginPosition(row(align, 1)),
             clippedEndPosition(row(align, 0)), clippedEndPosition(row(align, 1)));
-        extendSeed(seed, xDrop, score, seqs1[0], seqs2[0], 2 /* both */, GappedXDrop());
+        extendSeed(seed, seqs2[0], seqs1[0], 2 /* both */, score, xDrop, GappedXDrop());
 
         // find best global alignment of extended seed
         // TODO: Compute the best global alignment of extended seed and its score.
